src/functions.php

```diff
@@ -742,3 +751,16 @@ function parse($value)
 {
     return Context::get()->getEnvironment()->parse($value);
 }
+
+/**
+ * Throws a Exception when not called within a task-context.
+ *
+ * This method provides a usefull error to the end-user to make him/her aware
+ * to use a function in the required task-context.
+ */
+function requiresTaskContext($callerName)
```

Comment:
now I know why I decided against a method in Context ;-):

the message here is rather task specific and `Context` class atm is rather generic.

```
    /**
     * Throws a Exception when not called within a task-context.
     *
     * This method provides a usefull error to the end-user to make him/her aware
     * to use a function in the required task-context.
     *
     * @throws Exception
     * @return Context
     */
    public static function required($callerName)
    {
        $ctx = self::get();
        if (!$ctx) {
            throw new Exception("'$callerName' can only be used within a 'task()'-function!");
        }
        return $ctx;
    }
```

do you still think we should move it to Context, I am fine with any solution you will accept.